template/package.json

```diff
@@ -116,10 +120,9 @@
         "@semantic-release/git",
         {
           "assets": [
+            "./package.json",
             "./CHANGELOG.md",
             "./README.md",
-            "./README__labs.md",
```

Comment:
Why was the labs version removed here when the md file still exists in the repo?

Reply:
This file will never be in a created repo. This README file is used if the auroLabs template is requested.

https://github.com/AlaskaAirlines/WC-Generator/blob/302755b852358a1e1650a1604cbd17191d3ad72a/bin/generate.js#L170-L177